com.leohanney.Volaris.yml

```diff
@@ -0,0 +1,38 @@
+id: com.leohanney.Volaris
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+          
+command: volaris-gui
+rename-icon: com.leohanney.Volaris
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+
+          
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/volar-is/volaris-gui/releases/download/app-v0.1.0/Volaris_0.1.0_amd64.deb
+        sha256: c55febeee6cbde904b52119e86606dd08eb474aa8fc53be16843841334cdcc3c
+        only-arches: [x86_64]
+      - type: file
+        url: https://raw.githubusercontent.com/volar-is/volaris-gui/main/flatpak.xml
+        sha256: 61ce16a64ab7afb0221f0d943bd381817a6b5617e993106003ca899fa9ede748
+                  
+    build-commands:
+       - ar -x *.deb
+       - tar -xf data.tar.gz
+       - mkdir -p /app/bin
+       - cp -r usr/bin/* /app/bin
```

Comment:
you could just `mv usr/* /app/` and then use the `rename-` in the manifest to fix icons and desktop file names automatically.

(but building from source would be better.)